src/components/Header/components/PeopleSearch/index.js

```diff
@@ -15,7 +15,7 @@ import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import './people-search.css';
 import peopleSearch from '../../../../services/people-search';
-const MIN_QUERY_LENGTH = 3;
+const MIN_QUERY_LENGTH = 2;
```

Comment:
Yes, thank you for listening to my suggestion! This will make it possible for those with two letter first/last names to be effectively searched